OpenRA.Mods.Common/TargetExtensions.cs

```diff
@@ -68,8 +81,7 @@ public static Target Recalculate(this Target t, Player viewer, out bool targetIs
 						return Target.FromActor(t.FrozenActor.Actor);
 
 					// Original actor was killed while hidden
-					if (t.Actor == null)
```

Comment:
Can someone elaborate on this change? This block isn't related to bot visibility, and it looks like it will change behaviour for player-controlled units.

Reply:
See https://github.com/OpenRA/OpenRA/pull/17298#discussion_r341372098 above and the following discussion.

Reply:
That is fine for bots, but for players I expect that this was supposed to be `if (t.FrozenActor.Actor == null)`. The comment doesn't make sense if we are unconditionally returning `Target.Invalid`.

Reply:
Tbh, please revert the changes though. Every "contribution" from me has in the past caused problems and maintainers needed to clean up after me. It was the right decision for me to stop "contributing" since although I disagree with a lot of things, don't want to harm the project.

Reply:
Don't be too harsh with yourself, it is simply not true that everything you contributed was a problem.